service/project.go

```diff
@@ -59,10 +59,23 @@ func (uis *UIServer) projectsPage(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	opts := gimlet.PermissionOpts{
+		Resource:      evergreen.SuperUserPermissionsID,
+		ResourceType:  evergreen.ProjectResourceType,
```

Comment:
Admin permissions should be a separate permission type, otherwise there won't be any scope to match on

Reply:
Ah I did it so that there are three super user scopes (admin, project, and distro) all with the resource id `evergreen.SuperUserPermissionsID` and each with their respective resource type. I think the way you are thinking will require three different resource ids (one for each type of super user) and also three scopes (they would have to be separate scopes because we do not necessarily want a user to have all super user permissions if they are given one).

Reply:
But the 3 separate permissions take care of that. Why do we need more than 1 resource for superusers?

Reply:
you're right! sorry got a little confused, but the single resource is cleaner